src/screens/DiscoveryScreenV2/elements/DiscoveryTitleSeparator.js

```diff
@@ -1,36 +1,31 @@
 import * as React from 'react';
-import {Text} from 'react-native';
+import { Text } from 'react-native';
 
-import {fonts} from '../../../utils/fonts'
-import {COLORS} from '../../../utils/theme';
+import { colors } from "../../../utils/colors"
+import { fonts } from "../../../utils/fonts"
 
 /**
  * @typedef {Object} DiscoveryTitleSeparatorProp
  * @property {String} text
  */
 /**
- *
- * @param {DiscoveryTitleSeparatorProp} prop
+ * 
+ * @param {DiscoveryTitleSeparatorProp} prop 
  */
 const DiscoveryTitleSeparator = (prop) => {
-  const {text} = prop;
-  return (
-    <Text
-      style={{
+    const { text } = prop
+    return <Text style={{
         fontSize: 12,
         lineHeight: 18,
-        fontFamily: fonts.poppins[600],
-        color: COLORS.black,
+        fontFamily : fonts.poppins[600],
+        color: colors.black,
         paddingLeft: 20,
         paddingRight: 20,
         paddingTop: 11,
         paddingBottom: 11,
-        backgroundColor: COLORS.lightgrey
+        backgroundColor: colors.lightgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color property value has been changed. Make sure that the new color constant exists in the `colors` module and is the intended color.
```diff
- backgroundColor: COLORS.lightgrey
+ backgroundColor: colors.lightgrey,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->